ocs_ci/ocs/ui/base_ui.py

```diff
@@ -248,6 +248,16 @@ def get_elements(self, locator):
         """
         return self.driver.find_elements(by=locator[1], value=locator[0])
 
+    def get_element_attribute(self, locator, attribute, safe: bool = False):
+        """
+        Get attribute from WebElement
```

Comment:
Missing doc text on what is locator, attribute, safe

Reply:
done, thanks